core/assets/bundles/bundle_nl.properties

```
@@ -146,7 +146,7 @@ server.kicked.serverClose = Server afgesloten...
 server.kicked.vote = Je bent ge vote-kicked. Tot ziens.
 server.kicked.clientOutdated = Verouderde versie! Update jouw spel!
 server.kicked.serverOutdated = Verouderde server! Vraag de host om te upgraden!
-server.kicked.banned = Je bent verbannt van deze server.
+server.kicked.banned = Je bent gedegradeerd van deze server.
```

dat betekend lager in rang, ik zou voor deze gaan:
```suggestion
server.kicked.banned = Je bent verbannen van deze server.
```